helpdesk_mgmt/controllers/main.py

```diff
@@ -34,15 +34,17 @@ def support_ticket_close(self, **kw):
     def _get_teams(self):
         return (
             http.request.env["helpdesk.ticket.team"]
-            .sudo()
```

Comment:
Don't you need the sudo anymore?

Reply:
`sudo()` really isn't (and wasn't) necessary because there is no rule that filters records (except the multi-company rule).